src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  static removeAnimal(animal) {
+    delete this.alive[this.alive.indexOf(animal)];
+  }
+
+  constructor(name, health = 100) {
```

Comment:
```suggestion
  constructor(name) {
```
`health` always will be 100, you don't even need to pass it